pom.xml

```diff
@@ -6,9 +6,9 @@
     <version>1.46-SNAPSHOT</version>
     <packaging>pom</packaging>
     <properties>
-        <teamcity-version>8.1.5</teamcity-version>
-        <ci-sauce.version>1.132</ci-sauce.version>
-        <saucerest.version>1.0.32</saucerest.version>
+        <teamcity-version>9.1.6</teamcity-version>
+        <ci-sauce.version>1.144-SNAPSHOT</ci-sauce.version>
```

Comment:
We should probably release the downgraded plexus version as its own release and note that it's needed for teamcity.